Dockerfile

```diff
@@ -442,6 +442,32 @@ ENV PATH="/usr/libexec/rust/bin:$PATH" LD_LIBRARY_PATH="/usr/libexec/rust/lib"
 
 # =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=
 
+FROM sdk-libc as sdk-bootconfig
+
+USER root
+
+ARG KERNELVER="5.10.109"
```

Comment:
Maybe add a `TODO` to use the kernel sources from buildroot once they're new enough, after we've deprecated the 5.4 kernel variants.